src/osgPlugins/OpenFlight/GeometryRecords.cpp

```diff
@@ -35,1434 +36,1495 @@
 
 namespace flt {
 
-template<class ARRAY>
-void reverseWindingOrder( ARRAY* data, GLenum mode, GLint first, GLint last )
-{
-    switch( mode )
-    {
-    case osg::PrimitiveSet::TRIANGLES:
-    case osg::PrimitiveSet::QUADS:
-    case osg::PrimitiveSet::POLYGON:
-        // reverse all the vertices.
-        std::reverse(data->begin()+first, data->begin()+last);
-        break;
-    case osg::PrimitiveSet::TRIANGLE_STRIP:
-    case osg::PrimitiveSet::QUAD_STRIP:
-        // reverse only the shared edges.
-        for( GLint i = first; i < last-1; i+=2 )
-        {
-            std::swap( (*data)[i], (*data)[i+1] );
-        }
-        break;
-    case osg::PrimitiveSet::TRIANGLE_FAN:
-        // reverse all vertices except the first vertex.
-        std::reverse(data->begin()+first+1, data->begin()+last);
-        break;
-    }
-}
-
-void addDrawableAndReverseWindingOrder( osg::Geode* geode )
```

Comment:
Let's take the 3.6 version of this function since it is the same change we made just with different variable names.

Reply:
Reverted change to this function (pending)